plugins/woo-ai/src/product-name/suggestion-item.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * Internal dependencies
+ */
+import { AttributeSuggestion } from '../shared/types';
+
+const SuggestionItem = ( {
+	suggestion,
+	onSuggestionClick,
+}: {
+	suggestion: AttributeSuggestion;
+	onSuggestionClick: ( suggestion: AttributeSuggestion ) => void;
+} ) => (
+	<li className="suggestion-item">
+		<button
+			className="button select-suggestion"
+			type="button"
+			onClick={ () => onSuggestionClick( suggestion ) }
+		>
+			<p className="suggestion-content">{ suggestion.content }</p>
+			<p className="suggestion-reason">
```

Comment:
I think this suggestion reason is a great addition! Well done 👏 
Perhaps a designer is better suited to provide recommendations for this topic but I feel like people are trained to see the "info" icon and hover over it or click on it to get more information. My gut feeling is to get rid of the info icon. 🤔